changelogs/current.yaml

```diff
@@ -24,5 +24,8 @@ new_features:
     Change HTTP status to 200 to respect the gRPC protocol. This may cause problems for incorrect gRPC clients expecting the filter
     to preserve HTTP 1.1 responses.  This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.grpc_http1_reverse_bridge_change_http_status`` to false.
+- area: mobile
```

Comment:
I think you can skip - these are notes for envoy main and this is really a mobile-only PR